package.json

```diff
@@ -28,9 +28,9 @@
   },
   "devDependencies": {
     "@babel/core": "^7.17.8",
-    "@skyscanner/eslint-config-skyscanner": "^12.0.0",
+    "@skyscanner/eslint-config-skyscanner": "^16.0.0",
     "babel-jest": "^27.5.1",
-    "husky": "^7.0.0",
-    "jest": "^27.5.1"
+    "husky": "^8.0.3",
+    "jest": "^29.6.2"
```

Comment:
Our jest use in this repo is so light that we're not impacted

https://github.com/jestjs/jest/releases